Documentation/ApiOverview/Events/Events/Backend/ModifyPageLayoutContentEvent.rst

```diff
@@ -31,27 +35,3 @@ API
 ===
 
 ..  include:: /CodeSnippets/Events/Backend/ModifyPageLayoutContentEvent.rst.txt
-
-History / Migration
-===================
-
-The event :php:class:`TYPO3\\CMS\\Backend\\Controller\\Event\\ModifyPageLayoutContentEvent`
-has been introduced to serve as a more powerful and flexible alternative
-for the removed hooks
-:php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['cms/layout/db_layout.php']['drawHeaderHook']`
-and :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['cms/layout/db_layout.php']['drawFooterHook']`.
-
-In contrast to the removed hooks, this event does not provide the
```

Comment:
Should we really remove this information about the limitations of this Event?

Reply:
For other events we also do not mention which objects/variables have vanished with the change from hooks to events. It mentions that an object is not available anymore. This is common. And getting the page ID is now in the examples. The hook information is now at the beginning.